index.js

```diff
@@ -1,5 +1,6 @@
 function main() {
   console.log ('Hello world')
+  console.log ('Let\'s see this how this will be integrated');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/kentc-tt/sast-test/pullRequest?prid=6969844)
```suggestion
  console.log ("Let's see this how this will be integrated");
```